core/templates/domain/utilities/url-interpolation.service.ts

```diff
@@ -103,7 +107,8 @@ export class UrlInterpolationService {
    * If a URL requires a value which is not keyed within the
    * interpolationValues object, this will return null.
    */
-  interpolateUrl(urlTemplate: string, interpolationValues: any): string {
+  interpolateUrl(urlTemplate: string,
```

Comment:
Break after opening bracket.

Reply:
Doing this exceeds the length of the next line by 80 chars. We have to still place the second arg in the next line. Should I still do it?

Reply:
Yes, we follow the rule to break after opening bracket if break is needed.

Reply:
This comment isn't addressed.

Reply:
Done